WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt

```diff
@@ -1051,6 +1060,73 @@ class MySiteFragment : Fragment(),
         }
     }
 
+    @Subscribe(sticky = true, threadMode = ThreadMode.MAIN)
+    fun onEventMainThread(event: StorySaveResult) {
+        EventBus.getDefault().removeStickyEvent(event)
+        if (event.isSuccess()) {
+            // TODO WPSTORIES add TRACKS
+            // AnalyticsTracker.track(Stat.MY_SITE_ICON_UPLOAD_UNSUCCESSFUL);
+            // TODO WPSTORIES probably we want to remove this snackbar given we want to immediately start uploading it
+            val snackbarMessage = String.format(
+                    getString(string.story_saving_snackbar_finished_successfully),
+                    getStoryAtIndex(event.storyIndex).title
+            )
+            uploadUtilsWrapper.showSnackbar(
+                    requireActivity().findViewById<View>(R.id.coordinator),
+                    snackbarMessage
+            )
+        } else {
+            // TODO WPSTORIES add TRACKS
+            // AnalyticsTracker.track(Stat.MY_SITE_ICON_UPLOAD_UNSUCCESSFUL);
```

Comment:
Same thought as above.

Reply:
sure, refer to https://github.com/wordpress-mobile/WordPress-Android/pull/12074 👍